tests/components/generic_hygrostat/test_config_flow.py

```diff
@@ -0,0 +1,116 @@
+"""Test the generic hygrostat config flow."""
+
+from unittest.mock import patch
+
+from syrupy.assertion import SnapshotAssertion
+from syrupy.filters import props
+
+from homeassistant import config_entries
+from homeassistant.components.generic_hygrostat import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry, async_mock_service
+
+
+async def test_config_flow(hass: HomeAssistant, snapshot: SnapshotAssertion) -> None:
+    """Test the config flow."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result == snapshot(name="init", exclude=props("data_schema"))
+
+    with patch(
+        "homeassistant.components.generic_hygrostat.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {
+                "name": "My dehumidifier",
+                "dry_tolerance": 2,
+                "wet_tolerance": 4,
+                "humidifier": "switch.run",
+                "target_sensor": "sensor.humidity",
+                "device_class": "dehumidifier",
+                "away_humidity": 35,
+                "initial_state": True,
```

Comment:
can we use `CONF_*` for this?

Reply:
I've changed it. But for me i see a reason to avoid constants in tests, since if somebody where to change the value of this constant, the test would succeed, but users installs would crash.